storybook/storybook-docs/src/breakpoints.stories.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Design Guidelines/Breakpoints" />
+
+# Breakpoints
+
+We use 2 breakpoints where the design of a website adjusts for a good user experience: at screen widths of 576px and 1088px.
+
+We follow a mobile-first approach: first, we create the design below the breakpoint, and then we use a media query to create the design above the breakpoint.
+
+For example:
+
+```css
+.class {
+  /* Your narrow styling */
+
+  @media screen and (min-width: $amsterdam-breakpoint-wide) {
+    /* Your wide styling */
+  }
+}
+
+Within this repository, you can import the breakpoint as a Sass variable.
+You can find this variable in [packages/css/src/common/breakpoint.scss](https://github.com/Amsterdam/design-system/blob/main/packages/css/src/common/breakpoint.scss).
+```
```

Comment:
Are closing the syntax marks in the wrong place?

Reply:
Yes, fixed.